openapi/vosi-table-ops.yaml

```diff
@@ -0,0 +1,50 @@
+get:
+    operationId: table-ops-job-list
+    tags:
+      - Table Operations
+    summary: list async table update jobs
+    description: async job listing
+    parameters:
+    responses:
+        200:
+            $ref: ../uws/uws-responses.yaml#/job-listing
+        401:
+            $ref: ../vosi/vosi-std-responses.yaml#/not-authenticated
+        403:
+            $ref: ../vosi/vosi-std-responses.yaml#/permission-denied
+post:
+    operationId: table-ops-job-create
+    tags:
+      - Table Operations
+    summary: create async table update job 
+    description: TAP asynchronous table update (create UWS Job)
+    parameters:
+        - name: TABLE
+          in: query
+          description: a single table name as provided in tap_schema
+          required: true
+          schema:
+              type: string
+        - name: INDEX
+          in: query
+          description: create index operation; value is a column name in the table
+          required: false
+          schema:
+              type: string
+        - name: UNIQUE
+          in: query
+          description: qualifier for INDEX to create a unique index
+          required: false
+          schema:
+              type: boolean
+    responses:
+        303:
+            $ref: ../uws/uws-responses.yaml#/created
+        401:
+            $ref: ../vosi/vosi-std-responses.yaml#/not-authenticated
+        403:
+            $ref: ../vosi/vosi-std-responses.yaml#/permission-denied
+        404:
+            $ref: ./vosi-std-responses.yaml#/not-found
+        405:
+            $ref: ./vosi-std-responses.yaml#/not-implemented
```

Comment:
405 is not likely to be an appropriate response here - it's only correct if this resource exists but does not support any POST requests (and would have to be accompanied by an `Allow` header giving a list of HTTP methods that are supported for that resource) - see RFC9110 sec 15.5.6.

Reply:
Do you think that a 404 when probing for an optional endpoint is sufficient for the client? 

This is a UWS job list endpoint so I agree that optional means it is either there or not, but technically the possibility of a 405 is really up to UWS, which supports GET and POST to the job list and thus could respond with a 405 if a user tries other methods. Admittedly, that was not what I was thinking when I put it there (that was just cut&paste).